lookout/style/typos/preprocessing.py

```diff
@@ -0,0 +1,35 @@
+from typing import Set, Union
+
+import pandas
+
+from lookout.style.typos.utils import Columns
+
+
+def check_split(split: str, tokens_set: Set[str]) -> bool:
```

Comment:
`split.split()` looks strange to me.  consider renaming `split` to something else. For example, if it is a text line you can call it `line`, or just `text` or at least `to_split`

Reply:
Also, we do not use variable type in its name usually, so `tokens` totally file here I think. You do not have a list of tokens or any other structure.